core/chains/evm/config/toml/defaults/Linea_Sepolia.toml

```diff
@@ -0,0 +1,14 @@
+ChainID = '59141'
+FinalityDepth = 900
```

Comment:
This may need to be changed, discussion ongoing, but this one copied from the Goerli/Mainnet?

Reply:
This one is copied from CCIP where we are setting 1 hour finality on Linea. 
(Yes this could change after ongoing discussion about Linea's finality)